packages/react-components/src/components/styles/VideoGallery.styles.ts

```diff
@@ -182,3 +191,18 @@ export const localVideoCameraCycleButtonStyles: IButtonStyles = {
     background: 'transparent'
   }
 };
+
+/**
+ * Styles for the local video tile modal when it is focused, will cause keyboard move icon to appear over video
+ * @private
+ */
+export const localVideoModalStyles: IModalStyles = {
+  keyboardMoveIconContainer: {
+    zIndex: 6 // zIndex to set the keyboard movement Icon above the other layers in the video tile.
```

Comment:
always avoid raw zIndex values, this should be 
```
const keyboardMoveIconContainerZIndex = n + 1;
```
where n is an import from another file

Reply:
kk I think I know just the constant to use 😊